src/main/java/mServer/crawler/sender/orf/JsonUtils.java

```diff
@@ -2,6 +2,7 @@
 
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
+import de.mediathekview.mlib.tool.Log;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'de.mediathekview.mlib.tool.Log'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)